lighthouse-core/lib/strings.js

```diff
@@ -11,7 +11,9 @@ module.exports = {
   badTraceRecording: `Something went wrong with recording the trace over your page load. Please run Lighthouse again.`,
   pageLoadTookTooLong: `Your page took too long to load. Please follow the opportunities in the report to reduce your page load time, and then try re-running Lighthouse.`,
   pageLoadFailed: `Lighthouse was unable to reliably load the page you requested. Make sure you are testing the correct URL and that the server is properly responding to all requests.`,
+  pageLoadFailedInsecure: `The URL you have provided does not have a valid security certificate.`,
```

Comment:
hmm technically not just certificate, but admittedly most other security errors fall through the ERRORED_DOC_REQ path right now.  how about

> ... does not have valid security credentials.